intermediate_source/torch_export_tutorial.py

```diff
@@ -365,6 +426,29 @@ def forward(self, x, y):
 except Exception:
     tb.print_exc()
 
+######################################################################
+# We can also describe one dimension in terms of other.
```

Comment:
I think the restriction here is as a linear relation, A * Dim + B... not sure if there's more though. Avik would know best.